tools/check_format.py

```diff
@@ -71,8 +71,8 @@
 GRPC_INIT_WHITELIST = ("./source/common/grpc/google_grpc_context.cc")
 
 CLANG_FORMAT_PATH = os.getenv("CLANG_FORMAT", "clang-format-9")
-BUILDIFIER_PATH = os.getenv("BUILDIFIER_BIN", "$GOPATH/bin/buildifier")
-BUILDOZER_PATH = os.getenv("BUILDOZER_BIN", "$GOPATH/bin/buildozer")
+BUILDIFIER_PATH = (os.getenv("BUILDOZER_BIN") or (os.path.expandvars("$GOPATH/bin/buildifier") if os.getenv("GOPATH") else "Can't find buildifier binary"))
```

Comment:
can we just fallback to `shutil.which("buildifier")`? So if it exists in PATH we don't fail it.

Reply:
That sounds like a good idea.